README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
```

Comment:
Clarify what 'task install' specifically does, as users unfamiliar with this might not understand this step fully. Consider adding a brief description or a link to documentation for better user understanding.